### ISSUE TITLE
SnireIgnoreUnknownIV: True doesn't always work

### ISSUE THREAD

**Issue by Acrys** [PocketMobsters/PokeMobBot#450]
http://puu.sh/qnuGl/0ed250f438.png

Screenshot shows an unknown IV Blastoise coming in from feeder, it doesn't ignore it. This can happen pretty often.

**Comment by weedorbeat** [comment]
Its not a bug, the wiki says:

> **SnipeIgnoreUnknownIv (Value)**
... if PokeMobBot cannot determine the IV value from the sniping location service and this value is set to true, it will snipe them anyways. If set to false, PokeMobBot will ignore the Pokemon ...

In your config it is true, so it will catch unkown IV.

**Comment by Acrys** [comment]
I don't believe that is true, because with false it will snipe EVERYTHING that is unknown. When it is TRUE it will usually ignore a lot but still catch a lot.

**Comment by weedorbeat** [comment]
Unfortunately I have not tested it, looked up only in the wiki. Then the description in the wiki is wrong and the bug still exists, sorry.
